vercel.json

```diff
@@ -1,4 +1,5 @@
 {
+  "trailingSlash": true,
```

Comment:
moving to Pages means we lose the trailing slash, so I need to enable this globally. I don't anticipate any issues with this, given we enable it anyway on the origin urls.